pyOCD/target/cortex_m.py

```diff
@@ -942,6 +945,12 @@ def setBreakpoint(self, addr):
         if self.fpb_enabled is False:
             self.enableFPB()
 
+        if addr >= 0x20000000:
+            # Hardware breakpoints are only supported in the range
+            # 0x00000000 - 0x1fffffff on cortex-m devices
+            logging.error('Breakpoint out of range 0x%X', addr)
+            return False
```

Comment:
Where is this documented? I don't remember seeing this, and I was just reading through the FPB docs a couple days ago. Not to say I couldn't have missed it!

Does this also apply to watchpoints?